pt/news/_posts/2018-12-25-ruby-2-6-0-released.md

```diff
@@ -0,0 +1,143 @@
+
+* Adiciona uma `exception:` alternativa ao `Kernel#system` que causa uma exceção quando há uma falha, ao invés de retornar `false`. [[Feature #14386]](https://bugs.ruby-lang.org/issues/14386)
+
+* Adiciona o modo oneshot ao `Coverage` [[Feature#15022]](https://bugs.ruby-lang.org/issues/15022)
+
+  * Esse modo verifica "quando uma linha de código foi executado ao menos uma vez ou não", ao invés de "quantas vezes uma linha de código foi executada". Um gatilho por linha é disparado apenas uma vez, e uma vez que é disparado, a flag será removida, i.e., roda sem desperídicos.
+  * Adiciona `oneshot_lines:` keyword argument do Coverage.start.
+  * Adiciona `stop:` and `clear:` keyword arguments do Coverage.result. Se `clear` é verdadeiro, limpa os contadores para zero.  Se `stop` é verdadeiro, desabilita a metrificação de coverage.
+  * Coverage.line_stub é uma simples função de helper que cria o "stub" da cobertura da linha de um dado código-fonte.
+
+* Adiciona `FileUtils#cp_lr`.  Funciona como o cp_r mas cria links ao invés de cópias.  [[Feature #4189]](https://bugs.ruby-lang.org/issues/4189)
+
+## Melhorias de performance
+
+* Acelera o `Proc#call` por remover a alocação temporária para `$SAFE`.
+  [[Feature #14318]](https://bugs.ruby-lang.org/issues/14318)
+
+  Nós observamos uma melhoria de 1.4x na performance no teste de benchmark `lc_fizzbuzz`, que chama `Proc#call` diversas vezes. [[Bug #10212]](https://bugs.ruby-lang.org/issues/10212)
+
+*  Melhora a performance de `block.call` quando `block` é passado como o parâmetro de bloco. [[Feature #14330]](https://bugs.ruby-lang.org/issues/14330)
+
+  Combinado com melhorias nas operações com blocos do Ruby 2.5, a evaluação de blocos agora performa 2.6x mais rápido no teste de micro-benchmark com o Ruby 2.6. [[Feature #14045]](https://bugs.ruby-lang.org/issues/14045)
+
+*  Heap Transitente (`theap`) é introduzida. [[Bug #14858]](https://bugs.ruby-lang.org/issues/14858) [[Feature #14989]](https://bugs.ruby-lang.org/issues/14989)
+
+  `theap` é heap gerenciada para objetos de curta vida em memória, apontados para classes específicas (`Array`, `Hash`, `Object`, and `Struct`). Criando objetos Hash pequenos e de vida curta é 2x mais rápido. Com o teste de benchmark do rdoc, estatamos uma melhoria de 6-7% na performance.
+
+* Implementações nativas (`arm32`, `arm64`, `ppc64le`, `win32`, `win64`, `x86`, `amd64`) de co-rotinas para melhorar a troca de contextos com Fiber de forma significativa. [[Feature #14739]](https://bugs.ruby-lang.org/issues/14739)
+
+  `Fiber.yield` e `Fiber#resume` é até 5x mais rápido em Linux 64-bit. Programas que utilizam bastante do Fiber podem notar melhorias de performance maiores de 5% em geral.
+
+## Outras alterações notáveis desde a versão 2.5
+
+* `$SAFE` agora é um processo de estado global e pode ser setado para `0` novamente.  [[Feature #14250]](https://bugs.ruby-lang.org/issues/14250)
```

Comment:
Use single square brackets for links.

Reply:
`Setado` is not a Portuguese word.
Use `configurado` instead.

Reply:
```suggestion
* `$SAFE` agora é um processo de estado global e pode ser configurado para `0` novamente.  [[Feature #14250]](https://bugs.ruby-lang.org/issues/14250)
```